manuscript/algorithm.tex

```diff
@@ -181,3 +190,67 @@ \subsubsection{\gls{MARKAL}}
 of all the coefficients for the \gls{LP} problem, and is
 solved. 
 
+
+\subsection{Stochastic-Optimizing (SO)}
+Stochastic optimization refers to optimization methods
+that incorporate randomness in either the objective function
+or the constraints \cite{lauren_stochastic_2015}. It aims to find
+the roots for the objective function by generating random variables.
+This makes it fit for problems that contain uncertainty, which
+real world problems mostly are. 
+The random variables are defined by a probability distribution
```

Comment:
"The stochastic parameters driving the solution are typically sampled from probability distributions."